### ISSUE TITLE
Changes in loss values during training

### ISSUE THREAD

**Issue by RowanLiu97** [wubin5/STTN#7]
Can you explain why the loss value is oscillating during the training process rather than continuously decreasing (even if it increases, it should be a slight increase)?

**Comment by lixus7** [comment]
I think a scheduler method will be useful for the training fluctuation?

**Comment by zhaojiankuan** [comment]
1.调一下learning_rate和alpha会使loss更加稳定。
2.用的不是完整的数据集，训练效果会差一些。
